rpc/src/endpoint/net_info.rs

```diff
@@ -39,7 +39,7 @@ pub struct Response {
     pub n_peers: u64,
 
     /// Peer information
-    pub peers: Vec<PeerInfo>,
+    pub peers: Option<Vec<PeerInfo>>,
```

Comment:
I don't know, this is the existing domain type modeling.

Reply:
My only guess is, it could matter for serialization where the `peers` field can also have a null value (it cannot be missing/omitted from the struct because that would require `serde` attributes to allow). But I'd like to what's the purpose of this special case as a domain type and how does it interop. In [the Go implementation](https://github.com/tendermint/tendermint/blob/2ff11e5bc277db701b52ee07e74e81a2211150c4/rpc/core/types/responses.go#L119), the field type is an ordinary array, and it's specified as such in the OpenAPI manifest as well. Without more information, I'd suggest removing the `Option` wart for the 1.0 API.

Reply:
Cool, removing the `Option` sounds good.

Reply:
I suspect the Go code slips in a `null` instead of an array value somewhere, and this gets encoded into JSON accordingly. But I'd like to see a case where it's valid or otherwise has to be grandfathered in.